api/v1beta1/ssp_types.go

```diff
@@ -57,6 +58,26 @@ type SSPSpec struct {
 
 	// NodeLabeller is configuration of the node-labeller operand
 	NodeLabeller NodeLabeller `json:"nodeLabeller,omitempty"`
+
+	// DataImportCronTemplates defines a list of DataImportCrons managed by the SSP
+	// Operator. This is intended for images used by CommonTemplates.
+	DataImportCronTemplates []DataImportCronTemplate `json:"dataImportCronTemplates,omitempty"`
```

Comment:
Since this is related to common templates, can you move this field inside `CommonTemplates` struct?